src/AddTodoForm.js

```diff
@@ -1,10 +1,17 @@
 import React from "react";
 
-function AddTodoForm() {
+function AddTodoForm(props) {
+    function handleAddTodo(event) {
+        event.preventDefault();
+        const todoTitle = event.target.elements.title.value;
```

Comment:
While this does work, you don't actually need to dig into the `elements` property since you can skip straight to `title` and access its value.

```suggestion
        const todoTitle = event.target.title.value;
```